aws/resource_aws_autoscaling_group.go

```diff
@@ -78,6 +78,7 @@ func resourceAwsAutoscalingGroup() *schema.Resource {
 						"version": {
 							Type:         schema.TypeString,
 							Optional:     true,
+							Default:      "$$Default",
```

Comment:
I believe this fix should be the in test configuration and not in the resource. 😉 If I recall its a difference in Terraform 0.12's configuration handling regarding `$` escaping.

Reply:
I see so within the testing configuration we should not escape `$` and instead render the configuration such as `Version: "$Latest"`. Is that correct. I see that the mixed_instance_policy.launch_template.version` has an attribute of `Default: "$$Deault"` should that be removed at some point. Not within this PR.

Reply:
I broke this into a separate PR https://github.com/terraform-providers/terraform-provider-aws/pull/9488